src/hotspot/share/nmt/virtualMemoryTracker.hpp

```diff
@@ -25,16 +25,29 @@
 #ifndef SHARE_NMT_VIRTUALMEMORYTRACKER_HPP
 #define SHARE_NMT_VIRTUALMEMORYTRACKER_HPP
 
-#include "memory/allocation.hpp"
-#include "memory/metaspace.hpp" // For MetadataType
-#include "memory/metaspaceStats.hpp"
 #include "nmt/allocationSite.hpp"
-#include "nmt/nmtCommon.hpp"
+#include "nmt/vmatree.hpp"
+#include "nmt/regionsTree.hpp"
 #include "runtime/atomic.hpp"
-#include "utilities/linkedlist.hpp"
 #include "utilities/nativeCallStack.hpp"
 #include "utilities/ostream.hpp"
 
+// VirtualMemoryTracker (VMT) is the internal class of NMT that only the MemTracker class uses it for performing the NMT operations.
```

Comment:
`... uses it for ...`, delete "it" (grammar issue).

Reply:
Removed.